src/main/java/com/microsoft/jenkins/function/FunctionAppDeploymentCommandContext.java

```diff
@@ -72,7 +72,7 @@ public void configure(
         CommandService.Builder builder = CommandService.builder();
         if (isJava) {
             // For Java function, use FTP-based deployment as it's the recommended way
-            builder.withStartCommand(FTPDeployCommand.class);
+            builder.withStartCommand(ZipDeployCommand.class);
```

Comment:
zip is default, right?

Reply:
Yes, have discussed with PC, we will use Zip instead of FTP.